### ISSUE TITLE
Slow speed

### ISSUE THREAD

**Issue by Zhentao-Liu** [lzhnb/Analytic-Splatting#2]
I think it may be a little slower than original 3DGS. Have you ever tested its time efficiency?

**Comment by lzhnb** [comment]
Since we introduced more square root and exponentiation operations, the forward and backward passes will be slower. The same is true for the inference stage, and the CUDA implementation will be slower.
But after writing it in GLSL, the fragment shader can handle this part of the operation efficiently, so we are not very slow in the Online Viewer.
I have not yet found the reason why the tile-based rendering implemented in CUDA is slower than GLSL.

**Comment by Zhentao-Liu** [comment]
Thanks for your answer. Did you mean the training would be slower? And I try you code, its really much slower than the original 3DGS. The training time is very important to me. No offense.

**Comment by lzhnb** [comment]
Yes, I have tried to speed up the code, but many of the operations are unavoidable.
For efficiency, Mip-Splatting is indeed better than ours. I'm sorry that our method cannot meet your efficiency requirements.

**Comment by Zhentao-Liu** [comment]
Thanks for your guidance. Anyway, your work is valuable. Integral is more accurate than filtering.
